frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick_start_Access_a _code_snippet.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Add ability to filter Quick Starts catalog
+              As a user, i want to be able to filter Quick Starts in the catalog by status of the Quick Starts and search Quick Starts catalog by name, description, tags and prerequisites.
```

Comment:
Isn't this about copy to clipboard feature?

Reply:
Sorry about that, updated